Doc/library/sqlite3.rst

```diff
@@ -1591,6 +1574,22 @@ Cursor objects
       including :abbr:`CTE (Common Table Expression)` queries.
       It is only updated by the :meth:`execute` and :meth:`executemany` methods.
 
+   .. attribute:: row_factory
+
+      Control how a row fetched from this :class:`!Cursor` is represented.
+      Set to :class:`sqlite3.Row` or a custom :term:`callable`
+      that accept two arguments,
+      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
+      and return a custom object representing an SQLite row.
+      If ``None``, a fetched row is represented as a :class:`tuple`.
```

Comment:
```suggestion
      If ``None``, a fetched row is represented as the raw :class:`tuple`.
```

The current wording sounded just a touch awkward to me given the repetition from the previous. Maybe this makes the connection clearer and acknowledges the relationship?

Reply:
Hm, no, this is misleading, and I now see that the previous "raw" is misleading: a custom converter may have transformed a "raw string" to a FooBar object before the row factory is invoked, so there is nothing "raw" about the row results anywhere.

Reply:
Ah, okay—I was just using the same wording as previously used the describe the object passed to the row factory, as that (raw or otherwise) is what is returned directly if no custom row factory is set, no? In any case, that connection should be made clear, IMO.

Reply:
We could explain the "if None" case first, and then make the (raw) tuple connection when explaining the arg spec for custom row factories.

Reply:
Suggestion added here: https://github.com/python/cpython/pull/99507/files#r1030245938

Reply:
That seems to make sense to me and I was thinking I'd prefer the default behavior first; I'll take a closer look at your specific suggestion.